client/my-sites/github-deployments/deployments/index.tsx

```diff
@@ -47,6 +51,10 @@ export function GitHubDeployments() {
 	};
 
 	const renderContent = () => {
+		if ( isLoading ) {
```

Comment:
I think it's fine we show stale data while refetching. What problem were you trying to solve here?

Reply:
The UI appear glitchy, there was the loading placeholder , then the connection form flashes before showing the deployment list.

Reply:
In other words the `installations` finish loading before the deployments and that caused the connection form to be displayed. Then when the deployments finish loading the deployment list is shown.

Reply:
I see. That's a weird situation. I think the appropriate would be:

- display placeholder if no data
- load deployments while refreshing then hydate deployments or display connection form or authorization